pkg/entrypoint/entrypointer.go

```diff
@@ -80,6 +80,9 @@ type Entrypointer struct {
 	OnError string
 	// StepMetadataDir is the directory for a step where the step related metadata can be stored
 	StepMetadataDir string
+
+	//Dont Send results to the termination path.
+	DontSendResultsToTerminationPath bool
```

Comment:
Nit: negative variable name can be confusing 😅😅

I had some interesting reading about the negative boolean variable naming: 
- https://www.serendipidata.com/posts/naming-guidelines-for-boolean-variables
- https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig

Reply:
Thanks! I will look into it!

Reply:
I updated it based on Billy's suggestion. No more negative naming :)